sample-webtasks/es6-template-literals.js

```diff
@@ -0,0 +1,8 @@
+"use latest";
```

Comment:
Webtask runs node 4.4.x. This means there is no need for the `"use latest";` directive.